### PR TITLE
Stealthier Hacked Law amd Chameleon Projector (again)

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request
Chameleon Projector and Hacked AI Law traitor items have been changed to look like extremely suspicious but "plausibly deniable" contraband.

Projector is standard syndicate text check.
AI board requires science or the silicons to correctly identify.

## Why It's Good For The Game
The revert on the projector was knee jerk but this offers an alternative that encourages roleplay without just making the item completely innocent looking.

I started the reasons for the AI board change in my last PR. Security shouldnt be able to freely identify high level syndicate technology, but AI boards should be accessible in general.

## Changelog
:cl: Yakumochen
tweak: Chameleon Projector and the hacked AI board now require effort from security to completely identify but still appear as extremely suspicious contraband.
/:cl:

### CODE REVIEW

**Comment by CameronWoof** [comment]
These items are still immediately identifiable as what they actually are. This creates more opportunities for people to cry foul when they're confiscated (wah, mister admin he took my item called illegal tech, why he meta ban???) and makes no good use of the examine system.

While I'm not in favor of further buffing the chameleon projector (an item already regularly used to great effect with no way to counter it except to be banned for metagaming), I still think the AI board could be better implemented as a true stealth item if you really think it's too weak now.

**Comment by YakumoChen** [comment]
The point isnt to make the AI board stealthy, it's to give security a roleplay opportunity instead of just yelling "woop it's the hacked AI board perma time"

Security having a blue book of all traitor items is kind of dumb from an IC standpoint. They should at least require a modicum of effort to find out what black market, clandestine technology actually does instead of reading the label printed on every item.

**Comment by CameronWoof** [comment]
The point of the examine system was exactly, specifically to eliminate the issue where you knew what an item was out-of-character because of information the game gave you that our policy dictates your character isn't supposed to have.

Making "stealth" items unique and instantly identifiable out-of-character and expecting that players will get in trouble for correctly identifying them and acting accordingly completely defeats the purpose of having the examine system at all.

**Comment by YakumoChen** [comment]
The examine system can be used for purposes beyond just "completely hiding contraband from an ooc standpoint" and I've already proven as much with the PR I made before this.

Giving security opportunities to roleplay while not messing with item balance is completely sensible.

**Comment by CameronWoof** [comment]
"opportunities for roleplay" is not a good dogwhistle for "opportunities to make an admin angry and get jobbanned over our nonexistent contraband policy"

**Comment by YakumoChen** [comment]
From an OOC standpoint the projector never had ful protection in any iteration if these PRs because it still has a unique icon.

**Comment by YakumoChen** [comment]
The projector doesn't even have real meta protection in this PR regardless.  Nothing stops you from using it and immediately identifying it no matter what name you slap on it.

**Comment by CameronWoof** [comment]
It's not about the item, it's about its actual use case. If an officer does not physically see you activate the object, they are not allowed to (as an example of something I have literally been noted for) theorize that because you were reported in a room, you must still be in it, and baton the random cigarette butt in the middle of the room to reveal you.

The literal only rules-allowed way to catch someone using a champrojector is to have them activate it when you're in the room with them. No that's not a code issue, but buffing the item further by giving admins more reasons to pick on security officers who confiscate it only makes it worse.

**Comment by YakumoChen** [comment]
A stealth item being good for its purpose is not my issue. This PR does not grant or modify meta protection for either item, as they are still clearly identified as contraband of some kind.

**Comment by CameronWoof** [comment]
I reiterate that I hate this system being used to create banbait instead of remove it.

**Comment by YakumoChen** [comment]
I disagree. There is no sane case to be made for any sort of "metagame" behavior outside of outright naming what it is without testing it

**Comment by Grunnyyy** [comment]
It's not banbait, Cameron and I don't know why you are being so silly about it. 

If you hold it in your hand, you already got the valid. 

If you don't you can care less about it, since you don't know they have one


And if it's not metaprotected, I promise you, I will litter all of Maints with various trash, for the sole reason of having someone sift through tons of trash for meta gaming.

**Comment by CameronWoof** [comment]
As you can see above, our policy is not sane.

**Comment by Grunnyyy** [comment]
So, it can keep its examine as it will not change anything

You know it's illegal, even without it being used

And you can take it to what, science and have them be like ooohhh, funneh rp

You won't "loose". It's not always about winning.

**Comment by CameronWoof** [comment]
It's easy to spout platitudes like "it's not about winning" when you're using an item people will literally get banned for catching you using.

**Comment by Grunnyyy** [comment]
> It's easy to spout platitudes like "it's not about winning" when you're using an item people will literally get banned for catching you using.

If you see someone using it you are allowed to bust em for it

File: code/game/objects/items/AI_modules.dm [hunk 1]
```diff
@@ -464,9 +464,12 @@ AI MODULES
 /******************** Hacked AI Module ******************/
 
 /obj/item/ai_module/syndicate // This one doesn't inherit from ion boards because it doesn't call ..() in transmitInstructions. ~Miauw
-	name = "Hacked AI Module"
-	desc = "An AI Module for hacking additional laws to an AI."
+	name = "hacked circuitry" //SKYRAT CHANGE
+	desc = "A suspicious circuit board modified beyond recognition. It appears to be a module for bleeding edge technology." //SKYRAT CHANGE 
 	laws = list("")
+	special_desc_requirement = EXAMINE_CHECK_JOB // Skyrat edit
+	special_desc_jobs = list("Scientist, Roboticist, Research Director, Cyborg, AI") //SKYRAT CHANGE
```
Comment (Gandalf2k15) [inline code/game/objects/items/AI_modules.dm]:
```suggestion
	special_desc_jobs = list("Scientist", "Roboticist", "Research Director", "Cyborg", "AI") //SKYRAT CHANGE
```

**Comment by YakumoChen** [comment]
I really wish that example you made in your original PR wasn't so badly written Gandalf, it literally has commented 
`list(thing1, thing2)`
I ain't a coder, I write shit as written.

**Comment by YakumoChen** [comment]
@Gandalf2k15 what's holding this up?

**Comment by Gandalf2k15** [comment]
Seems good to me.

**Pull request merged**